app.py

```diff
@@ -30,29 +30,41 @@ def _annotate(self, mmif: Union[str, dict, Mmif], **parameters) -> Mmif:
             docs = mmif.get_documents_by_type(DocumentTypes.VideoDocument)
         conf = self.get_configuration(**parameters)
         whisper_model = self.whisper_models.get(conf['modelSize'], None)
+        whisper_language = conf['modelLang']
```

Comment:
One thing that was discussed in a meeting (but never documented in #17) was to allow users to pass two-tier language code in `<lang>-<region>` format, e.g., `en-US` or `en-GB`, or `zn-CN`). In such a case, we need to pass just the first part to whisper and keep the second part, putting them together back in `@language` field in the text document output.